api/v1beta1/azurecluster_default_test.go

```diff
@@ -139,7 +139,7 @@ func TestVnetDefaults(t *testing.T) {
 							},
 						},
 						NodeOutboundLB: &LoadBalancerSpec{
-							FrontendIPsCount: to.Int32Ptr(1),
+							FrontendIPsCount: pointer.Int32Ptr(1),
```

Comment:
I see that there are some uses of `pointer.Int32Ptr` and `pointer.Int32Deref`. The PR description mentions that `pointer.Int32Deref` is being used to replace `to.Int32Ptr`. Will this be a problem? I noticed that `pointer.Int32Deref` is used for cases where the int might be nil, so I think it is the correct usage.

Reply:
> `pointer.Int32Deref` is being used to replace `to.Int32Ptr`

Actually it's like this:

`to.Int32Ptr` -> `pointer.Int32` (return a pointer / reference)
`to.Int32` -> `pointer.Int32Deref` (return an int32 or 0)

Those changes *should* preserve identical behavior.

Reply:
```
// Int32Ptr is a function variable referring to Int32.
// Deprecated: Use Int32 instead.
var Int32Ptr = Int32 // for back-compat
```

seems we should be using `Int32` though? `Int32Ptr` is deprecated

Reply:
Oops, yes, that's a mistake that crept in through a rebase: `pointer.Int32` is preferred now. I'll update.